docs/FRSKY_R9.md

```diff
@@ -252,12 +252,12 @@ Note: Python3 is a prerequisite to use the ELRS UARTUpload.py script.
 
 2. Download or git clone the [ExpressLRS](https://github.com/ExpressLRS/ExpressLRS) repository. If you have subversion installed: Since you only need the src/python folder, you could use the svn github interface to get just what you need. From the command line type something like this: "svn export https://github.com/ExpressLRS/ExpressLRS/tree/master/src/python"
 
-3. Connect a USB-TTL serial adapter to your computer and install a driver if it is not recognized.  
+3. Connect a USB-TTL serial adapter to your computer and install a driver if it is not recognized. Some CP210X based adapters have been found to not work, if you have one and are encountering errors please use a different USB-TTL adapter.  
 - Note: If you are having trouble flashing or don't have a USB-TTL serial adapter, you should be able to use your flight controller instead - see step 8.
 
 4. Wire the USB-TTL serial adapter to your receiver as instructed [here](https://www.expresslrs.org/quick-start/receivers/r9/#wiring-up-your-receiver). But, don't connect the VCC wire yet.
 
-5. Use the command window to run the UARTupload.py script like: "python UARTupload.py image\_file\_path" where image\_file\_path is the path to the appropriate .elrs file you obtained in step 1.
+5. Use the command window to run the UARTupload.py script by navigating to the folder where that script is located in the command window such as: "C:\Users\XXX\Downloads\ExpressLRS-master\ExpressLRS-master\src\python\UARTupload.py image\_file\_path" where image\_file\_path is the path to the appropriate .elrs file you obtained in step 1. If the upload fails by connecting to the wrong COM port, at the end of the firmware name add the appropriate COM port. For example, if you cloned the ExpressELRS repo into downloads and the firmware was also in downloads the appropriate windows command line would look like the following: C:\Users\XX\Downloads\ExpressLRS-master\ExpressLRS-master\src\python>uartupload.py C:\Users\XX\Downloads\rx-R9MM-f103rb-elrs-bl-v1.2.00.elrs COMXX
```

Comment:
I guess I wouldn't try flashing this way :) how about instead:

'Move the .elrs firmware file to the src/python folder downloaded in step 2.  Open a command line processor and navigate to the src/python folder.  Then execute the command 'python UARTupload.py XYZ.elrs' where XYZ is the .elrs firmware file for your hardware.' 

- Note: If the upload fails by connecting to the wrong port, you can specify the appropriate port after the firmware filename, e.g. 'python UARTupload.py XYZ.elrs COM11'

Reply:
that's indeed even better 👍

Reply:
I do not know how to edit this commit sorry not very versed in github.